cookbook/default-snippets.rst

```diff
@@ -1,62 +1,64 @@
-How to define a default-snippet?
-================================
+Using default snippets for managing page-independent data
+=========================================================
 
-Sulu gives the content-manager an easy way to define default snippets per type or area.
-These default-snippets will be used if the snippet-selection is
-empty on a specific page. This default selection will not be displayed in
-the form and can have a big impact on the page.
+The snippet feature of Sulu allows to create and manage reusable pieces of content.
+These pieces of content cannot only be assigned to a page using the ``snippet_selection``
+content type, but they can also be associated to a webspace.
 
-Defining areas
---------------
+To do this, Sulu allows to define **snippet areas** which then can be assigned to a
+**default snippet** on the ``Snippet Area`` tab of the webspace in the administration
+interface. After assigning default snippets to snippet areas via the administration
+interface, Sulu provides different solutions for loading the data of the configured snippet
+for a specific area and webspace.
 
-Sulu allows you to to reuse the same snippet template to use it as default
-for different snippet content types or load a snippet by its area key with
-the twig extension. The areas can be defined in the xml template of the snippets.
+This functionality is especially helpful to allow the content manager to managing
+webspace-wide data such as social media links or contact information.
+
+Defining possible areas for a snippet type
+------------------------------------------
+
+Each snippet template can define multiple snippet areas. All defined snippet areas will
+be displayed on the ``Snippet Area`` tab of the webspace in the administration interface
```

Comment:
This tab should be something like  `Areas` or `Snippet Areas` but sadly currently here still the name `Default Snippets` is used.

Reply:
not really related with this PR, but why dont you like `Default Snippets`? 🙂

Reply:
Because that wording should since introduciton of the areas not longer exist sulu/sulu#3402